src/services/todo.services/todoServices.ts

```diff
@@ -0,0 +1,39 @@
+import getToDos from "../../repository/todo.repository/todoRepository";
+import { NewToDoDto, ToDo } from "../../repository/todo.repository/types";
+import { v4 as uuidv4 } from "uuid";
+
+export function validateBody(body: NewToDoDto) {
+  if (!body) {
+    return { error: "Missing body" };
+  }
+  if (!body.title) {
+    return { error: "Missing title" };
+  }
+  if (!body.description) {
+    return { error: "Missing description" };
+  }
+}
+
+export function createToDoObject(title: string, description: string) {
```

Comment:
Naming - no need for the `Object` word inside of the function name - createToDo would be enough.

Also, the return type for this function is missing.